rewrite-java-17/src/main/java/org/openrewrite/java/isolated/ReloadableJava17ParserVisitor.java

```diff
@@ -1339,42 +1339,68 @@ private TypeTree annotatedTypeTree(Tree node, Map<Integer, JCAnnotation> annotat
         } else if (node instanceof JCAnnotatedType) {
```

Comment:
I was wondering if we need this whole if block or if it is covered by the `convert()`. If it is required, then I wonder if a test case is missing, as a breakpoint on the `convert()` statement never hit when running the rewrite-java-tck tests. Even when running all the rewrite-java-test tests the breakpoint was also just hit twice, but removing this block doesn't seem to cause any problems.

Reply:
I tested this and this block isn't necessary anymore -- I'll remove it in a moment